lib/internal/util.js

```diff
@@ -431,6 +433,39 @@ function createDeferredPromise() {
   return { promise, resolve, reject };
 }
 
+let internalTimers;
+
+function getActiveResources() {
+  if (internalTimers == null) {
+    internalTimers = require('internal/timers');
+  }
```

Comment:
If we don't need backporting to v14:
```suggestion
  internalTimers ??= require('internal/timers');
```

Reply:
I don't know yet whether this should be backported to v14. Maybe more reviews will help resolve that. :)